sklearn-demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    np.random.seed(0)
Line #6.    cov1 = np.array([[0., -0.1], [1.7, .4]])
</pre><p>This is how you can comment on lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>